extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,92 +1,81 @@
 <div class="wds-global-navigation">
 	<div class="wds-global-navigation__content-bar">
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-logo">
+		<a href="<?= Sanitizer::encodeAttribute( $model['logo']['header']['href'] ); ?>" class="wds-global-navigation__logo">
 			<?= DesignSystemHelper::getSvg(
-				'wds-company-logo-fandom',
-				'wds-global-navigation__content-item-logo-fandom'
+				$model['logo']['header']['image'],
+				'wds-global-navigation__logo-fandom'
 			) ?>
-			<span class="wds-global-navigation__content-item-logo-powered-by">powered by</span>
+			<span class="wds-global-navigation__logo-powered-by">powered by</span>
```

Comment:
We are going to use a single SVG for this whole section. It's already returned by API in https://github.com/Wikia/app/pull/11235/files#diff-d8454a9e4a0c5b8fa20a138ae8351f3cR20 and I've created https://wikia-inc.atlassian.net/browse/XW-1945 to add the asset to DS.